### PR TITLE
CI cleanup

### PR DESCRIPTION
**Pull request by walro**
See commit messages for more info.

### CODE REVIEW

File: .github/workflows/ci-windows.yml [hunk 1]
```diff
@@ -20,10 +24,10 @@ jobs:
       - name: Install dependencies
         run: |
           nuget restore Twingly.Search.sln
-          nuget install NUnit.Console -Version 3.11.1 -OutputDirectory packages
+          cinst nunit-console-runner --version=3.11.1 --yes
```
Comment (walro) [inline .github/workflows/ci-windows.yml:27]: OK, this installation is super slow for some reason. Locally it's superfast. I'll see if something can be done, it looks like chocolatey just stalls for some reason

Comment (walro) [reply .github/workflows/ci-windows.yml:27]: Snippet from logs:

```
Fri, 18 Dec 2020 09:15:19 GMT
Chocolatey v0.10.15
Fri, 18 Dec 2020 09:17:30 GMT
Installing the following packages:
Fri, 18 Dec 2020 09:17:30 GMT
nunit-console-runner
```

Not sure what that ~2min wait time is all about.

Comment (walro) [reply .github/workflows/ci-windows.yml:27]: I am guessing it pulls a list of packages, I also noted that we have the same behavior over at Arvegods, it stalls for 1 minute or so there too.

Comment (walro) [reply .github/workflows/ci-windows.yml:27]: I will just revert to the old behavior (i.e. installing via NuGet)

Comment (walro) [reply .github/workflows/ci-windows.yml]: Done (via force push)

**Review by Pontus4 — APPROVED** [review]
LGTM!

**Pull request merged**